public/app/ui/components/collections/library/list-result.html

```diff
@@ -0,0 +1,4 @@
+<div class="list-result list-result-library">
+    <span class="pull-left" data-bind="component: displayFor('MediaThumbnail', result)"></span>
+    <a href="#" data-bind="attr: { href: detailUrlFor(result) }, component: displayForLabelField(result)"></a>
```

Comment:
We also need the following fields below the title:
```
^Author <br/>
^Publisher: ^PlaceOfPublication, ^DateOfPublication.
```
I'd also argue the ^PublicationType should also be included.  Maybe presented like a tag and on the right, but @mostro20 is probably best placed to make that call.